PDF_UA/WCAG-21-Complete.xml

```diff
@@ -132,7 +132,7 @@
                 <reference specification="ISO 32000-1" clause="14.8.5.4.3"/>
             </references>
         </rule>
-        <rule object="SALinkAnnotation">
+        <rule object="SALinkAnnotation" tags="major,human,structure,annotation,alt-text">
```

Comment:
I assume this is also machine. We have a deterministic algorithm to check for alt-test, that is not a URL. Whether it is meaningful or not is human. But we don't check this